src/store/routeAssets/getters.ts

```diff
@@ -115,18 +116,37 @@ const getters = defineGetters<RouteAssetsState>()({
           'symbol'
         )
       ).map((assetArray: Array<Recipient>) => {
+        const reduceData = assetArray.reduce(
+          (acc, item) => {
+            return {
+              usd: new FPNumber(item.usd).add(acc.usd),
+              total: new FPNumber(item.amount || 0).add(acc.total),
```

Comment:
better to use predefined static value from library

`item.amount ? new FPNumber(item.amount) : FPNumber.ZERO`...